doc/classes/AnimationLibrary.xml

```diff
@@ -56,6 +56,7 @@
 	</methods>
 	<members>
 		<member name="_data" type="Dictionary" setter="_set_data" getter="_get_data" default="{}">
+			Contains a [Dictionary] of [Animation] resources.
 		</member>
```

Comment:
Seems like a bug that a member starting with an underscore is exposed.

Sounds like this was just meant to serialization, not public access?

Reply:
I think it would be fine. It needs to have its usage changed to `PROPERTY_USAGE_NO_EDITOR | PROPERTY_USAGE_INTERNAL`. If it needs to be exposed, it needs to be renamed anyway. So we'd be breaking compatibility.

If we do neither, we risk it being accidentally unexposed if we, say, fix the fact that an underscored member is added to the docs (it should be stripped from XMLs by the doctool, the fact that it's not is a bug).

Reply:
I made https://github.com/godotengine/godot/pull/84376